### ISSUE TITLE
DataFrame.sort_values doesn't accept str for `by` param

### ISSUE THREAD

**Issue by tekumara** [microsoft/pylance-release#144]
## Environment data

-   Language Server version: v2020.7.2
-   OS and version: macOS
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.6.10

## Expected behaviour

No type error, see https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sort_values.html

## Actual behaviour

```
No overloads for "df.sort_values(by='X')" match parameters
  Argument types: (Literal['X'])Pylance (reportGeneralTypeIssues)
```

## Code Snippet / Additional information

```python
from pandas import DataFrame

def sort(df: DataFrame) -> DataFrame:
    return df.sort_values(by='X')
```

**Comment by gramster** [comment]
Should be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.8.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202081-13-august-2020
